boards/beagle/beaglev_fire/beaglev_fire_common.dtsi

```diff
@@ -10,22 +10,10 @@
 #include <mem.h>
 
 / {
-	model = "BeagleV-Fire";
+	model = "beagle,beaglev-fire";
```

Comment:
Why is the model converted to a compatible syntax?

Reply:
This is the format recommended by the Devicetree Specification (section 2.3 "Standard properties", point 2.3.2 "model"):
```
The recommended format is: "manufacturer,model", where manufacturer is a string describing
the name of the manufacturer (such as a stock ticker symbol), and model specifies the model number.
```